src/pageObject.js

```diff
@@ -1,16 +1,42 @@
-'use strict';
+"use strict";
 
 class Header {
+  get logo() {
+    return "Conduit logo";
+  }
+}
 
-};
-
-class PageObject {
+class PageObject extends Header {
```

Comment:
Inheritance should be used when there is an 'is-a' relationship between classes. In this case, it seems like `PageObject` is not a type of `Header`. Instead, `PageObject` has a `Header`, which implies a 'has-a' relationship. Consider using composition instead of inheritance here.